lib/plan_executor/executor.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
+
+# Used for $ERROR_INFO. This *must* be capitalized!
+require 'English'
+require 'json'
+require 'logging'
+require 'set'
+require 'bolt/result'
+require 'bolt/config'
+require 'bolt/transport/api'
+require 'bolt/notifier'
+require 'bolt/result_set'
+require 'bolt/puppetdb'
+
+module PlanExecutor
+  class Executor
+    attr_reader :noop, :transport
+
+    def initialize(noop = nil,
+                   bundled_content: nil,
+                   load_config: true)
+      @bundled_content = bundled_content
+      @logger = Logging.logger[self]
+      @plan_logging = false
+      @load_config = load_config
```

Comment:
this is only used by the ssh transport so it should probably be left off.  I think it's just a weird way to test ssh.